dist/jstat.js

```diff
@@ -144,20 +144,49 @@ jStat.dimensions = function dimensions(arr) {
 };
 
 
-// Returns a specified row as a vector
+// Returns a specified row as a vector or return a sub matrix by pick some rows
 jStat.row = function row(arr, index) {
+  if(isArray(index)){
+	  return index.map(function(i){
+		  return jStat.row(arr,i);
+	  })
+  }
   return arr[index];
 };
 
+jStat.rowa=function (arr,i){
```

Comment:
Mind adding a comment with a brief description about what it does or returns? Same with any other new functions that don't have one.